traco.gemspec

```
@@ -11,9 +11,9 @@ Gem::Specification.new do |s|
   s.summary     = "Translatable columns for Rails 4.2 or better, stored in the model table itself."
   s.license     = "MIT"
 
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- spec/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
+  s.files         = Dir["{lib}/**/*"] + %w[LICENSE.txt]
```

How do you feel about this version, from the linked article? Either is fine by me, though if we keep `%w[ foo ]` I think it should have spaces per our styleguide :)

```suggestion
  s.files         = Dir["{lib}/**/*", "LICENSE.txt"]
```

---

Your edit is nicer!